.github/workflows/pycafe-dashboards-in-CI.yml

```
@@ -0,0 +1,47 @@
+name: PyCafe Dashboards in CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches:
+      - main
```

```suggestion
on:
  push:
    branches: [main]
  pull_request:
    branches:
      - main
    paths:
      - "vizro-core/**"
```

To only run the job when edits to vizro-core are made. Not that it takes extra processing to run it, but probably reduces spamming from the bot commenting on every PR.